core/src/main/java/org/infinispan/conflict/impl/NullValueEntry.java

```diff
@@ -0,0 +1,22 @@
+package org.infinispan.conflict.impl;
+
+import static org.infinispan.commons.util.Util.toStr;
+
+import org.infinispan.container.entries.ImmortalCacheEntry;
+
+/**
+ * @author Ryan Emerson
+ * @since 9.1
+ */
+class NullValueEntry extends ImmortalCacheEntry {
```

Comment:
Yes. Previously I couldn't, however I have altered my approach so that it's now possible.